eslint/config.js

```diff
@@ -160,4 +160,14 @@ module.exports = {
 		'valid-typeof': 'error',
 		yoda: 'off',
 	},
+	overrides: [
+		{
+			files: 'packages/**/*.js',
+			settings: {
+				react: {
+					pragma: 'createElement',
+				},
+			},
+		},
+	],
```

Comment:
I believe we still need a similar config for babel to generate the Right JS (instead of using the global)

Reply:
Confirmed, I will look into it.

Reply:
It should be fixed with f7afc46.